docs/supported_platforms_and_configurations.md

```diff
@@ -37,7 +37,7 @@ For more information see [conan-io/conan-docker-tools](https://github.com/conan-
 ## Windows
 
 - Compilers: Visual Studio:
-  - 2015 (19.0.24215)
+  - 2015 (null)
```

Comment:
Very interesting.

I guess that is technically correct :laughing:

Reply:
😓  Indeed, we are no longer generating binaries for VS2015

This automation needs some work

Reply:
Well, at least is not NaN